tests/conftest.py

```diff
@@ -75,10 +75,6 @@ def restart_savepoint(session, transaction_in):
 @pytest.fixture(autouse=True)
 def client(request, db_session):
     """Provide a test client."""
-    if 'noclient' in request.keywords:
-        # To use this, annotate a test with:
-        # @pytest.mark.noclient
-        return None
```

Comment:
This doesn't work because if there is a `yield` statement in a fixture, there can't be a `return` or a second `yield`. This fixture will have to be rewritten without using `yield` to allow it to disable itself.